### ISSUE TITLE
ESP-01S Istallation Issue

### ISSUE THREAD

**Issue by KL-XL** [bigtreetech/BIGTREETECH-TFT35-V3.0#67]
Hi there,

I am a complete newbie in 3d printers. Just got the BTT TFT35 E3 V3.0, two ESP-01S modules and BTT Writer Module from BIGTREETECH Global Store at AliExpress. The TFT35 installed and operates smoothly, no issues. Having trouble installing ESP-01S. I followed all the steps in the manual:
- successfully flashed firmware using Writer Module, esp3d.bin file and flash_download_tools_v3.6.7 software
- downloaded the three files for upload: 404.htm, favicon.ico and index.html.gz
- placed ESP-01S on the TFT35 board
- got the IP 192.168.0.1
- connected my laptop to ESP3D network using the password 12345678
- opened 192.168.0.1 in MS Edge and got the initial file upload interface
- uploaded the three files and pressed refresh, the files appeared in the uploaded list (so far so good)
Here is where I got the issue: the Configuration Wizard didn't pop up. I waited for a while and the refreshed the page. The page turned blank. Now no matter what browser I use the 192.168.0.1 always shows blank. If I open it in IE then I can see the favicon in the address bar which sort of confirms that I see the right page and that the files have uploaded. However I am unable to progress nor revert to the initial file upload interface. The module still shows the IP when I turn on the printer and I can connect to ESP3D network no problem.
I am clueless as to if the modules are faulty or it a software issue. 
Any guidance is highly appreciated?

Thanks,
KL

My configuration:
machine: Ender 3
board: BTT SKR Mini E3 V2.0
display: BTT TFT35 E3 V3.0
OS: Windows 10
slicer: Cura 4.6.1
other: Pronterface/Printrun 1.6.0
![Display](https://user-images.githubusercontent.com/60302316/85048832-29a24d00-b194-11ea-8604-1b7679b22a6b.jpg)
![Installation page](https://user-images.githubusercontent.com/60302316/85048834-2ad37a00-b194-11ea-8a77-c02dae0a6498.png)
![wifi](https://user-images.githubusercontent.com/60302316/85048835-2ad37a00-b194-11ea-83af-b2766e5c5535.png)

**Comment by oldman4U** [comment]
Hi.

I guess you uploaded a firmware which was too big for your Espies module. So i would recommend to reset the board like described here:
<img width="652" alt="Screenshot 2020-06-21 at 00 56 49" src="https://user-images.githubusercontent.com/54359396/85213107-4f794e80-b35a-11ea-8ac6-a478c6149640.png">

Once you are back to the page, where you can upload firmware, take the firmware from here as a starting point: https://github.com/bigtreetech/BIGTREETECH-SKR-PRO-V1.1/tree/master/firmware/esp-01s

Let me know

**Comment by KL-XL** [comment]
Thank you oldman4U. I am able to see the upload page now. So it is a great relief :) It says the module is 92% full. The firmware you provided the link for is the one I used originally. I tried to upload it and update via this upload page but I get the message "Update Cancelled". Appreciate your further guidance.
KL

![Upload Page](https://user-images.githubusercontent.com/60302316/85213606-f2cd6200-b360-11ea-8d98-a2d08760d557.png)
![Firmware Error](https://user-images.githubusercontent.com/60302316/85213609-f82aac80-b360-11ea-8fb8-867f8ee91a18.png)
